gateway.py

```diff
@@ -10,7 +10,8 @@
 import socket
 import servlet
 import pprint
-from collections import OrderedDict, defaultdict
+from typing import Any, Dict, List, Set, Tuple  # noqa
```

Comment:
Please specify exactly which error in this places so that we don't potentially silence useful errors.

Reply:
* `typing` imports are only ever used by `mypy` and are unused as far as `flake8` is concerned.
* `try: import foo except: import foobar as foo` is detected as re-assignment.
* Certain modules are just not recognized by `mypy`.

Reply:
no, I don't mean like that. the # noqa syntax lets you specify which error you're ignoring instead of all of them